crosswalks/Citation_File_Format_1.2.0.README.md

```diff
@@ -0,0 +1,183 @@
+# Crosswalk for Citation File Format 1.2.0
+
+## Format information
+
+Citation File Format (CFF) is a YAML format for providing citation metdaata for software (and experimentally for datasets).
+
+- Website: <https://citation-file-format.github.io/>
+- Specifications: <https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md>
+- Schema: <https://github.com/citation-file-format/citation-file-format/blob/main/schema.json> (JSON Schema)
+
+## Crosswalk specifics
+
+The following sections provide details for directional mapping 
+in cases where mapping is not bidirectionally unambiguous.
+
+### `downloadUrl` <=> `repository-artifact`
+
+- CodeMeta `downloadUrl`: "If the file can be downloaded, URL to download the binary."
+- CFF `repository-artifact`: "The URL of the work in a build artifact/binary repository."
+
+While the scope of `repository-artifact` is documented to be somewhat narrower than `downloadUrl`, 
+its main use case is discovery (e.g. for reuse and reproducibility),
+which is arguably the same as for `downloadUrl`.
+Therefore the two can be assumed to signify the same.
+
+Future versions of CFF may document this more clearly (https://github.com/citation-file-format/citation-file-format/issues/326).
+
+### `softwareRequirements`/`softwareSuggestions` <=> `references`
+
+- CodeMeta `softwareRequirements`: "Required software dependencies"
+- CodeMeta `softwareSuggestions`: "Optional dependencies , e.g. for optional features, code development, etc"
+- CFF `references`: "Reference(s) to other creative works."
+
+#### CodeMeta to CFF
+
+The scope of `references` could be documented more clearly in CFF (https://github.com/citation-file-format/citation-file-format/issues/327).
```

Comment:
```suggestion
The scope of `references` could be documented more clearly in CFF (https://github.com/citation-file-format/citation-file-format/issues/327).
```
```suggestion
The scope of `references` could be documented more clearly in CFF (<https://github.com/citation-file-format/citation-file-format/issues/327>).
```

Reply:
Done in 70ea2df3f2f1ebe4425fe12a5c928554f8937dbe.